cg/cli/workflow/commands.py

```diff
@@ -89,6 +91,29 @@ def link(context: CGConfig, case_id: str, dry_run: bool):
     analysis_api.link_fastq_files(case_id=case_id)
 
 
+@click.command("metrics-deliver")
+@ARGUMENT_CASE_ID
+@OPTION_DRY
+@click.pass_obj
+def metrics_deliver(context: CGConfig, case_id: str, dry_run: bool) -> None:
```

Comment:
I don't know if this should be in the general commands or the nf_analysis.py. So far it seems that is only the NF pipelines using it. So I think I would move it there

Reply:
I moved it to `cg/cli/workflow/nf_analysis.py` instead